.buildkite/hooks/pre-command

```diff
@@ -28,6 +28,10 @@ fi
 echo "Creating var/ artifact directory..."
 docker run --rm -v /tmp/artifacts:/artifact-mount alpine:latest /bin/sh -c 'chown -R 2000 /artifact-mount/' || true
 
+if [[ "${OSTYPE}" == msys ]]; then
+    powershell -Command "msiexec.exe /i https://awscli.amazonaws.com/AWSCLIV2.msi /qn"
```

Comment:
should this be built into the AMI?

Reply:
I guess the missing windows awscli on the AMI might be the reason that docker plugin does not work with ecr?

also, have you tested if this works?

does this run before the docker plugin or after the docker plugin?